### PR TITLE
Fixes and some cleaning done to language translations.

### PR DESCRIPTION
**Pull request by nikomakela**
PT-830. Fixes after testing #92 .

Screenshots: https://helsinkisolutionoffice.atlassian.net/browse/PT-830?focusedCommentId=23605

### CODE REVIEW

File: src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx [hunk 1]
```diff
@@ -10,9 +10,6 @@ interface Props {
 const OccurrenceGroupInfo: React.FC<Props> = ({ occurrence }) => {
   const { t } = useTranslation();
 
-  const languages = occurrence.languages.edges.map(
-    (edge) => edge?.node?.id ?? ''
-  );
```
Comment (nikomakela) [inline src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx:15]: This wasn't used.

File: src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const EnrolmentForm: React.FC<Props> = ({
   const { options: studyLevelOptions } = useStudyLevels();
 
   const languageOptions = Object.values(Language).map((level) => ({
-    label: translateValue('enrolment:language.', level, t),
```
Comment (nikomakela) [inline src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx:45]: Using common translations.

File: src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupLanguageInfo.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ const OccurrenceGroupLanguageInfo: React.FC<Props> = ({ occurrence }) => {
     <p>
       {t('occurrence:labelLanguages')}:{' '}
       {languages
-        ?.map((language) => t(`occurrence:languages.${language}`))
```
Comment (nikomakela) [inline src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupLanguageInfo.tsx:20]: Using common translations.

File: public/static/locales/fi/common.json [hunk 4]
```diff
@@ -5,12 +5,12 @@
...
-    "fi": "Suomi",
-    "ru": "Venäjä",
-    "sv": "Ruotsi",
-    "zh_hans": "Kiina"
+    "ar": "arabia",
+    "en": "englanti",
+    "fi": "suomi",
+    "ru": "venäjä",
+    "sv": "ruotsi",
+    "zh_hans": "kiina"
```
Comment (nikomakela) [inline public/static/locales/fi/common.json:13]: Languages should be small in Finnish by default. Can be capitalized in code if needed.

**Review by Sartonon — APPROVED** [review]

**Pull request merged**